hdk-rust/src/api.rs

```diff
@@ -901,9 +906,32 @@ pub fn query(entry_type_name: &str, start: u32, limit: u32) -> ZomeApiResult<Que
     }
 }
 
-/// Not Yet Available
-pub fn send(_to: Address, _message: serde_json::Value) -> ZomeApiResult<serde_json::Value> {
-    Err(ZomeApiError::FunctionNotImplemented)
+/// Sends a node-to-node message to the given agent.
+/// This works in conjunction with the `receive` callback that has to be defined in the
+/// [define_zome!](macro.defineZome.html) macro.
+///
+/// This functions blocks and returns the result returned by the `receive` callback on
+/// the other side.
```

Comment:
I want to add example code in here to this description, but I am open to / happy to do that as a followup

Reply:
Great 👍